.github/workflows/ci.yml

```diff
@@ -212,6 +212,16 @@ jobs:
         # erroring about invalid credentials instead.
         if: github.event_name == 'push' || env.DEPLOY == '1' || env.DEPLOY_ALT == '1'
 
+      - name: upload job metrics to DataDog
+        if: needs.calculate_matrix.outputs.run_type != 'pr'
+        env:
+          DATADOG_SITE: datadoghq.com
+          DATADOG_API_KEY: ${{ secrets.DATADOG_API_KEY }}
+          DD_GITHUB_JOB_NAME: ${{ matrix.name }}
+        run: |
+          npm install -g @datadog/datadog-ci
```

Comment:
should this `npm install` be pinned in order to prevent new version breakage?

maybe something like this `npm install -g @datadog/datadog-ci@2.38.x` or `npm install -g @datadog/datadog-ci@^2.x.x`

Reply:
Good point, I was also thinking about that. If we do pin, the version should stay the same (even though transitive dependencies can still be updated without a lockfile), but the command might eventually no longer be supported by DataDog (meaning its internal API can change), so it might also break our CI in the future. If we don't pin, then it can break if the external CLI changes.

I'm fine with pinning.